pinot-core/src/main/java/org/apache/pinot/core/util/SchemaUtils.java

```diff
@@ -79,6 +81,8 @@ public static void validate(Schema schema) {
     for (FieldSpec fieldSpec : schema.getAllFieldSpecs()) {
       if (!fieldSpec.isVirtualColumn()) {
         String column = fieldSpec.getName();
+        Preconditions.checkState(!StringUtils.containsWhitespace(column),
+            "The column name \"%s\" should not" + " contain blank space.", column);
```

Comment:
(nit) Can be "should not contain blank space" instead of "should not" + " contain ...."